pulp_ansible/app/models.py

```diff
@@ -482,7 +490,22 @@ def last_sync_task(self):
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
-        permissions = (("modify_ansible_repo_content", "Can modify ansible repository content"),)
+        permissions = [
+            ("rebuild_metadata_ansiblerepository", "Can rebuild metadata on the repository"),
+            ("repair_ansiblerepository", "Can repair the repository"),
+            ("sign_ansiblerepository", "Can sign content on the repository"),
+            ("sync_ansiblerepository", "Can start a sync task on the repository"),
+            (
+                "copy_collection_version_ansiblerepository",
+                "Can copy collection versions on the repository",
+            ),
+            (
+                "move_collection_version_ansiblerepository",
+                "Can move collection versions on the repository",
+            ),
+            ("manage_roles_ansiblerepository", "Can manage roles on repositories"),
+            ("modify_ansible_repo_content", "Can modify repository content"),
```

Comment:
@newswangerd What do you think about these separate permissions for copy and move? They seem to overlap with modify, but modify is also being used for upload.

Reply:
I would just use the permissions for modify for copy and move. I would prefer to keep the permissions relatively primitive. Copy and move are just instances of modify and view